src/primitive.js

```diff
@@ -1316,6 +1316,9 @@ Mavo.observe({id: "primitive"}, function({node, type, attribute, record, element
 		else if (attribute === "mv-options") {
 			node.updateOptions();
 
+			let option = node.options.get(node.value) ?? node.options.keys().next().value;
```

Comment:
The default value will be taken into account inside the node anyway, so this is not the right place to deal with it, and it would be an undersirable coupling (now every time we touch default values, we also need to look in the code that deals with `mv-options` updates). 

In fact, I wonder if even this line might be duplicating logic that should be elsewhere. Isn't there already logic that sets the value appropriately based on `node.options`? If so, we should be invoking that…

Reply:
@LeaVerou, I could manage to find a place where, from my perspective, this logic should be. Could you please give this PR another shot?